server/package.json

```
@@ -0,0 +1,26 @@
+{
+    "name": "prisma-vscode-server",
+    "description": "Implementation of a language server in node.",
+    "version": "0.0.31",
+    "author": "Prisma",
+    "license": "Apache-2.0",
+    "engines": {
+        "node": "*"
```

Maybe like all our packages like this: 
```suggestion
        "node": ">=10"
```